### ISSUE TITLE
Brown Street Piano Keys Idea

### ISSUE THREAD

**Issue by jeremywight** [hackgvl/public-art#1]
@pamelawoodbrowne @ajhodges @allella @bikeoid  So after our discussion on Tuesday night I had an idea, to use ARKit as a means to make this easier and also more immersive. I’ve included a drawing and also a free hand which is a collaborative drawing app if you’re interested in ideating on it with me.

![9fca5ede-76dc-4b42-85ed-5b15a8298e8c](https://user-images.githubusercontent.com/8245600/42032429-86d3ebd0-7aa7-11e8-8552-56c33d69355e.png)

Freehand: https://freehand.invisionapp.com/freehand/document/cUerNEubv

**Comment by bikeoid** [comment]
Interesting concept - it could possibly even expand to other AR activities downtown.   But this would still be  good as something unique to Greenville.

**Comment by invisionjeremy** [comment]
@bikeoid Yeah I think it could be kind of fun as well. See what kind of immersive experiences we could create

**Comment by pamelawoodbrowne** [comment]
Here's the [PDF version of the application](https://www.greenvillesc.gov/DocumentCenter/View/4917/Application-for-Placement-of-Public-Art?bidId=) for Art in Public Places with a contact person listed.

**Comment by invisionjeremy** [comment]
Email sent to the contact person

**Comment by allella** [comment]
Rodney Trusty https://www.facebook.com/rodney.trusty is working on interactive stuff and interested in the idea.  You guys should reach out to him and try to connect.
